portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsPreview.testcase

```diff
@@ -367,14 +367,43 @@ definition {
 	}
 
 	@description = "Verify that a Text Field can be previewed"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test TextFieldCanBePreviewed {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133783 TextFieldCanBePreviewed pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		FormFields.setPlaceholder(fieldValue = "Test Placeholder Text");
+
+		FormFields.editHelpTextField(fieldHelpText = "Test Help Text");
+
+		FormsAdminNavigator.gotoPreviewForm();
+
+		Navigator.selectSpecificWindow(windowTitle = "Form - Forms");
+
+		FormFields.viewTextField(
+			fieldHelpText = "Test Help Text",
+			fieldLabel = "Text",
+			fieldName = "Text");
+
+		Form.viewFormPlaceholderText(key_value = "Test Placeholder Text");
+
+		FormFields.editText(
+			fieldName = "",
```

Comment:
@piresdm, shouldn't we use the Text Field name here?

Reply:
@natocesarrego It doesn't need, but I included it anyway!